SiteAssets/pages/3.0/scripts/CDM.DATA.CONTROLLER.js

```diff
@@ -729,10 +729,30 @@ tposdata = {
 
         hListItem.update();
         ctx().executeQueryAsync();
-        if (callback) {
+        if (callback && callback === Function) {
```

Comment:
This throws an error when call back is not a function hence the second condition